fabric-networking-api-v1/src/main/java/net/fabricmc/fabric/api/networking/v1/ServerConfigurationNetworking.java

```diff
@@ -224,6 +224,19 @@ public static void send(ServerConfigurationNetworkHandler handler, CustomPayload
 
 	// Helper methods
 
+	/**
+	 * Gets the brand of the client the player connected with.
+	 *
+	 * @param handler the network handler, representing the connection to the player/client
+	 * @return client's brand, or {@code null} if not sent
+	 */
+	@Nullable
+	public static String getBrand(ServerConfigurationNetworkHandler handler) {
```

Comment:
What are your toughts on moving this out into its own PR? Will make things a bit easier to review, this change can likely go in quite quickly.

Reply:
It would also be nice to have a way to get the servers brand.

Reply:
Sure thing